fg_lib/README.rst

```diff
@@ -0,0 +1,22 @@
+.. _fg_lib:
+
+Nordic Fuel Gauge library (FG lib)
+##################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The fuel gauge library is the software complement of battery measurements made by a Nordic Power Management IC (PMIC).
+Measurements made during device operation are processed by this library, which results in a state-of-charge (SOC) prediction, along with other metrics.
```

Comment:
```suggestion
The fuel gauge library processes battery measurements made by Power Management ICs (PMICs) and provides a state-of-charge (SOC) prediction, along with other metrics.
The library is meant for use with devices from the Nordic nPM Family of PMICs.
```
This "software complement" sounds weird to me -- like something that is required, but obviously the usage of the library is optional. Also, I think we can merge both sentences.